indigo_app/forms/works.py

```diff
@@ -1025,6 +1043,17 @@ def filter_queryset(self, queryset, exclude=None):
 
         return queryset.distinct()
 
+    def facet_item(self, field, value, count):
+        try:
+            label = next(lab for (val, lab) in self.fields[field].choices if val == value)
+        except StopIteration:
+            raise ValueError(f"Unknown choice {value} for field {field}")
```

Comment:
does this error need translation?

Reply:
no, it's not user facing, it will raise a 500-level error. It's to guard against typos lower down.